service/controller/v5/resource/blobobject/desired.go

```diff
@@ -20,14 +22,29 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	storageAccountName := key.StorageAccountName(customObject)
+	clusterCerts, err := r.certsSearcher.SearchCluster(key.ClusterID(customObject))
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
 
+	storageAccountName := key.StorageAccountName(customObject)
 	containerName := key.BlobContainerName()
+	certificateEncryptionSecretName := key.CertificateEncryptionSecretName(customObject)
+
+	encrypter, err := r.toEncrypterObject(ctx, certificateEncryptionSecretName)
+	if apierrors.IsNotFound(err) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "encryptionkey resource is not ready")
+		resourcecanceledcontext.SetCanceled(ctx)
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
```

Comment:
[fmt](https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#control-flow) says the log comes first. Below is another occasion of this.